hr_attendance_modification_tracking/models/hr_attendance.py

```diff
@@ -17,3 +19,33 @@ class HrAttendance(models.Model):
     check_out = fields.Datetime(
         track_visibility='onchange'
     )
+    time_changed_manually = fields.Boolean(
+        string="Time changed",
+        required=False,
```

Comment:
```suggestion
```
default required is false

Reply:
Well noted. 
Will be updated in next commit, just now.
Thanks.